### ISSUE TITLE
helloworld.html

### ISSUE THREAD

**Issue by pallavikotturu** [pallavikotturu/mywebsite#1]
<!DOCTYPE html>
<html lang="en">
<head>
  <title>Pallavi Kotturu</title>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
    <script src="http://ajax.googleapis.com/ajax/libs/jqueryui/1.10.1/jquery-ui.min.js" type="text/javascript"></script> 
    
    <link href="//maxcdn.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css" rel="stylesheet">
    
     <link rel="stylesheet" href="style.css">
    
<meta property="og:image" content="/Users/pallavi/Desktop/IMG_0519.jpg"/>

 <nav class="navbar  navbar-inverse  navbar-fixed-top">
  <div class="container">
  <button type="button" class="navbar-toggle"
  data-toggle="collapse"
  data-target=".navbar-collapse"
  >
  <span class="sr-only"> Toggle navigation</span>
  <span class="icon-bar"> </span>
  <span class="icon-bar"> </span>
  <span class="icon-bar"> </span>
  </button>
  
      <a class="navbar-brand"> <font color = "lightgrey" face="Didot">P|KOTTURU</font></a>
   
       <div class="navbar-collapse collapse">
           <ul class="nav navbar-nav navbar-right">
               <li class=""><a href="about.html"><font color = "lightgrey" size=1>ABOUT</font></a> </li>
               <li> <a href="workExp.html"> <font color = "lightgrey" size=1>WORK EXPERIENCE</font></a> </li>
               <li> <a href="motivate.html"><font color = "lightgrey" size=1>MOTIVATIONAL CONTENT </font></a> </li>
               <li> <a href="thinkabout.html"><font color = "lightgrey" size=1>THINGS TO THINK ABOUT</font></a> </li>
               <li> <a href="influence.html"><font color = "lightgrey" size=1>INFLUENCERS</font></a> </li>
		   </ul>
       </div>
  </div>
</nav>
    
     <style>
            div {
                padding: 0px 100px 0px;
            }
        </style>
   
              <header>
        <div align="center">
            <br/> <br/> <br/>
            <h1><font face="Didot" color=black size=48>P A L L A V I . K O T T U R U </font></h1>
            <h4><font face="Didot" color=black> Leader | Visionary | Motivational Speaker | Technologist </font></h4>
            <br/> 
        </div>
      </header>
    
    <center><img src="/Users/pallavi/Desktop/IMG_0519.jpg" width=1300 height= 900px alt="" Style="border-top: 12px outset silver; border-bottom: 12px outset silver; border-left: 0px outset silver; border-right: 0px outset silver;" ></center>
        <div class="voffset3">
            <div class="row-md-8 row-sm-4">
                <br/> <br/> <br/> <br/> 
                <h3><font face="Didot Bold" align=center><stong>Welcome to Pallavi's website!</stong></font> </h3>
                <h4><font face="Didot Bold">Pallavi is an Indian-born college student who currently 
                attends Temple University in Philadelphia, PA. This website is designed to give people a greater insight 
                of the person that she is by presenting information about her career and work experiences, motivational material
                developed by her, things that she thinks about dealing with the world and universe at large, and a list of the key influencers from her life experiences. </font></h4>
                <br/> 

                    <h4><font face="Didot Bold">Drop Pallavi a Line: </font></h4>

    
                    <h4><font face="Didot Bold">Have a question or comment? Fill out the following information and Pallavi will get back to you as soon as possible.</font></h4>
                    <hr>
    
    <form class="form-horizontal" method="post" role="form">
    	<div class="form-group">
            <label for="email" class="col-sm-2 "><font face="Didot">E-mail</font></label>
    		<div class="col-sm-4">
    			<input type="email" class="form-control" id="email" name="email" placeholder="E-mail Address" required autofocus>
    		</div>
    	</div>
    	
    	<div class="form-group">
            <label for="name" class="col-sm-2"><font face="Didot">Name</font></label>
    		<div class="col-sm-4">
    			<input type="text" class="form-control" id="name" name="name" placeholder="Name" required>
    		</div>
    	</div>
    	
    	<div class="form-group">
            <label for="name" class="col-sm-2"><font face="Didot">Message</font></label>
    		<div class="col-sm-6">
    			<textarea id="message" name="message" class="form-control" placeholder="Your Message" rows="5" required></textarea>
    		</div>
    	</div>
    
    	<div class="form-group">
    		<div class="col-sm-offset-2 col-sm-6">
                <button type="submit" class="btn btn-default"><font face="Didot" Color="maroon">Send Message</font></button>
                <br/>
    		</div>
    	</div>
    </form>
                    <br/>

</div>
                <br/>
                <br>
            </div>
  

  <div class="navbar  navbar-default"> 
     <div class="container">
       <div class="navbar-text">
	    <p> <font face="Didot Bold">© P.Kotturu 2016</font></p>
	   </div>
	      
	 </div>
   </div>

  
</html>
